src/Deprecations.jl

```diff
@@ -47,5 +47,7 @@ end
 
 # Deprecated in 0.16.*
 
-@deprecate map_coeffs(f) map_coefficients(f)
+@deprecate map_coeffs(f, g::AbstractAlgebra.PolyElem; parent=parent(g)) map_coefficients(f, g; parent=parent)
```

Comment:
This also has a `cached` keyword argument I think (Oscar tests are failing because it seems to be missing).

Reply:
Yeah I went by the documentation, which was wrong.